infra/dataform-export/storage.js

```diff
@@ -0,0 +1,33 @@
+const { Storage } = require('@google-cloud/storage')
+const { Readable } = require('stream')
+
+const storage = new Storage()
+
+class StorageExport {
+  constructor (bucket) {
+    this.bucket = bucket
+  }
+
+  async exportToJson (data, fileName) {
```

Comment:
Report JSONs are currently non-gzipped.
We could compress it in the new pipeline, but will it have an impact?

Some of files are up to 1 MB.